StyleCop.Analyzers/StyleCop.Analyzers/Helpers/TriviaHelper.cs

```diff
@@ -153,7 +153,7 @@ internal static SyntaxTriviaList WithoutTrailingWhitespace(this SyntaxTriviaList
         internal static SyntaxTriviaList WithoutLeadingWhitespace(this SyntaxTriviaList triviaList, bool endOfLineIsWhitespace = true)
         {
             var nonWhitespaceIndex = IndexOfFirstNonWhitespaceTrivia(triviaList, endOfLineIsWhitespace);
-            return (nonWhitespaceIndex >= 0) ? SyntaxFactory.TriviaList(triviaList.Take(nonWhitespaceIndex)) : SyntaxFactory.TriviaList();
+            return (nonWhitespaceIndex >= 0) ? SyntaxFactory.TriviaList(triviaList.Skip(nonWhitespaceIndex)) : SyntaxFactory.TriviaList();
```

Comment:
:warning: **Behavior change:** I have no idea how we missed this before, but `WithoutLeadingWhitespace` was actually returning *only* the leading whitespace...